docs/related/async-raml-datatypes.mdx

```diff
@@ -0,0 +1,324 @@
+---
+id: async_raml_datatypes
+title: Async + RAML Data Types in AMF
+---
+
+:::info Technical Document
+This is a technical document, this information is intended for experienced users
+:::
+
+Within the Async specification it is possible to use raml data types to define a payload, taking into account that the schema format is specified accordingly.
+Both specifications have their own distinct referencing and modularization mechanisms, and it is not clear where to draw the line to determine where one specification stops, and the other one starts.
+This document will describe how to use RAML types in Async when using AMF.
+
+
+## Referencing external RAML content using ‘$ref’
+In order to support referencing raml content in external files, AMF allows the use of the `$ref` mechanism present in Async but with clear restrictions.
+This enables referencing data type fragments, data types present in libraries, as well as any raml content present in yaml or json files.
+
+Given that the payload content must comply with RAML, **the ‘$ref’ may only be used at the root of the payload definition as an exception to this rule**.
```

Comment:
What is "this rule" referring to? The payload complying with RAML? We should make this clearer.

Reply:
Hello Chris! Yes, "this rule" is refering to the compliance with RAML.
We can make this explicit by rephrasing to: "the ‘$ref’ may only be used at the root of the payload definition as an exception to the RAML specification"

Reply:
Thank you both for your feedback, I've made the corresponding changes.